Symfony/CS/Fixer/Contrib/HeaderCommentFixer.php

```diff
@@ -51,6 +53,36 @@ public static function getHeader()
     }
 
     /**
+     * Sets the newline mode.
+     *
+     * The multiline comment will appear with a leading newline by default.
+     * This setting will control whether the newline appears or not.
+     *
+     * @param bool $new Should the newline appear?
+     */
+    public static function setUseLeadingNewLine($new)
+    {
+        self::$useLeadingNewLine = (bool) $new;
+    }
+
+    /**
+     * Sets the comment mode.
+     *
+     * The multiline comment is by-default rendered with /*. This setting
+     * allows for the usage of docblock comments /**.
+     *
+     * @param bool $new Should the comment be a docblock comment?
+     */
+    public static function setUseDocBlockComment($new)
+    {
+        self::$useDocBlockComment = (bool) $new;
```

Comment:
This shouldn't need casting given that in the phpdoc you said only `bool` was allowed.